src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerCFTTest.java

```diff
@@ -243,7 +243,7 @@ public void should_return_a_200_with_task_warnings() {
             .and().contentType(MediaType.APPLICATION_JSON_VALUE)
             .and().body("task.id", equalTo(taskId))
             .body("task.warnings", is(true))
-            .body("task.permissions.values", hasItems("Read","Refer","Own","Manage","Cancel"));
+            .body("task.permissions.values", hasItems("Read","Refer","Own"));
```

Comment:
as per requirement, we will need to change the matcher to check this contains only the permissions relevant to the user